bin/db-initialization.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
```

Comment:
It's a good idea to make scripts executable and add a shebang so they can be run directly.

It also seems like we need some documentation in this script explaining how to run it.